pkg/network/usm/ebpf_ssl.go

```diff
@@ -51,38 +51,38 @@ var openSSLProbes = []manager.ProbesSelector{
 					EBPFFuncName: "uretprobe__SSL_write_ex",
 				},
 			},
-		},
-	},
-	&manager.AllOf{
-		Selectors: []manager.ProbesSelector{
 			&manager.ProbeSelector{
 				ProbeIdentificationPair: manager.ProbeIdentificationPair{
-					EBPFFuncName: "uprobe__SSL_do_handshake",
+					EBPFFuncName: "uprobe__SSL_connect",
```

Comment:
Why envoy doesn't contain SSL_connect,set_fd symbols ?
That could possibly break https in some other situation or envoy use only BIO ?